From 05e10a940ca734df90684059963b10bd0a832584 Mon Sep 17 00:00:00 2001
From: Eric Ruei <e-ruei1@ti.com>
Date: Sat, 25 Aug 2018 16:24:10 -0400
Subject: [PATCH] qt5_broswer: use webkit/browser instead of
 webengine/demobrowser

Upstream_Status: Inappropriate [other]
- This patch is a tentative workaround until the webengine web page
  display problem can be fixed.

Signed-off-by: Eric Ruei <e-ruei1@ti.com>
---
 qt5_apps/qt5_browser/qt5_browser.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qt5_apps/qt5_browser/qt5_browser.desktop b/qt5_apps/qt5_browser/qt5_browser.desktop
index a5d3953..2f6636f 100644
--- a/qt5_apps/qt5_browser/qt5_browser.desktop
+++ b/qt5_apps/qt5_browser/qt5_browser.desktop
@@ -4,7 +4,7 @@
 Name=Browser
 GenericName=Demo App
 Icon=/usr/share/matrix-gui-2.0/apps/images/web-icon.png
-Exec=/usr/share/qt5/examples/webenginewidgets/demobrowser/demobrowser http://www.ti.com
+Exec=/usr/share/qt5/examples/webkitwidgets/browser/browser http://www.ti.com
 Type=Application
 ProgramType=gui
 Categories=Qt5
-- 
1.9.1

